frontend/packages/console-dynamic-plugin-sdk/docs/console-extensions.md

```diff
@@ -953,23 +953,23 @@ Adds a new storage class provisioner as an option during storage class creation.
 
 | Name | Value Type | Optional | Description |
 | ---- | ---------- | -------- | ----------- |
-| `CSI` | `ProvisionerDetails` | yes |  |
-| `OTHERS` | `ProvisionerDetails` | yes |  |
+| `CSI` | `ProvisionerDetails` | yes | Constainer Storage Interface provisioner type |
+| `OTHERS` | `ProvisionerDetails` | yes | Others provisioner type |
 
 ---
 
 ## `console.storage-provider`
 
 ### Summary 
 
-(not available)
+Allows contributing new storage provider to select when attaching a storage along with provider specific component.
```

Comment:
I think this needs wordsmithing a bit for clarity. I am not sure I captured it correctly:
```suggestion
This extension can be used to contribute a new storage provider to select when attaching storage and a provider specific component.
```